src/main/java/org/myrobotlab/service/Agent.java

```diff
@@ -904,8 +932,14 @@ public Process spawn(CmdOptions inOptions) throws IOException, URISyntaxExceptio
     Platform platform = Platform.getLocalInstance();
     String exeName = platform.isWindows() ? "javaw" : "java";
     pd.javaExe = String.format("%s%sbin%s%s", System.getProperty("java.home"), fs, fs, exeName);
-
-    pd.jvm = new String[] { "-Djava.library.path=libraries/native", "-Djna.library.path=libraries/native", "-Dfile.encoding=UTF-8" };
+    
+    String jvmArgs = "-Djava.library.path=libraries/native -Djna.library.path=libraries/native -Dfile.encoding=UTF-8";

```

Comment:
it'd be nice to be able to override j(ava|ni).library.path  variables...  (as a future enhancement.)  seems like you should be able to override/append to the library paths (sometimes that's useful.)